activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -197,4 +224,24 @@ def emulating_unavailability
       Dalli.send(:remove_const, :Server)
       Dalli.const_set(:Server, old_server)
     end
+
+    def servers(cache = @cache)
+      client(cache).instance_variable_get(:@servers)
+    end
+
+    def client(cache = @cache)
+      cache.instance_variable_get(:@data)
+    end
+
+    def with_memcache_servers_environment_variable(value)
+      original_value = ENV["MEMCACHE_SERVERS"]
+      ENV["MEMCACHE_SERVERS"] = value
+      yield
+    ensure
+      if original_value.nil?
+        ENV.delete("MEMCACHE_SERVERS")
```

Comment:
This might not be needed, but I figured it would make sense to remove it as a key entirely if it wasn't there originally.